### PR TITLE
Run google-java-format on GCS NIO

### PR DESCRIPTION
**Pull request by jart**
I also made the following additional changes:
- Don't use one-line javadocs, per request of @aozarov
- Use public domain John Keats poetry published in 1819
- Minor cleanup to a few JavaDocs

### CODE REVIEW

**Comment by jart** [comment]
Ping @jean-philippe-martin

**Comment by jean-philippe-martin** [comment]
LGTM, and no dig against Keats but why remove Hamlet?
Surely it's old enough to be public domain?

**Comment by jart** [comment]
Hamlet was in two files. So I removed the duplication :)

File: gcloud-java-contrib/gcloud-java-nio/src/main/java/com/google/gcloud/storage/contrib/nio/UnixPath.java [hunk 1]
```diff
@@ -57,7 +57,9 @@ private UnixPath(boolean permitEmptyComponents, String path) {
     this.permitEmptyComponents = permitEmptyComponents;
   }
 
-  /** Returns new UnixPath of {@code first}. */
+  /**
+   * Returns new UnixPath of {@code first}.
```
Comment (ajkannan) [inline gcloud-java-contrib/gcloud-java-nio/src/main/java/com/google/gcloud/storage/contrib/nio/UnixPath.java]: Nit: could use {@code UnixPath} here

File: gcloud-java-contrib/gcloud-java-nio/src/main/java/com/google/gcloud/storage/contrib/nio/CloudStorageObjectImmutableException.java [hunk 2]
```diff
@@ -1,7 +1,10 @@
 package com.google.gcloud.storage.contrib.nio;
 
-/** Exception thrown to indicate we don't support a mutation of a cloud storage object. */
+/**
+ * Exception thrown to indicate we don't support a mutation of a cloud storage object.
```
Comment (ajkannan) [inline gcloud-java-contrib/gcloud-java-nio/src/main/java/com/google/gcloud/storage/contrib/nio/CloudStorageObjectImmutableException.java]: Super nit: we can capitalize "Cloud Storage" (it's lower case in a couple other places, so maybe a search-replace might be helpful)

**Comment by ajkannan** [comment]
It seems like there are a couple missed one/two-liner javadocs in CloudStorageConfiguration (for example [here](https://github.com/GoogleCloudPlatform/gcloud-java/blob/gcs-nio/gcloud-java-contrib/gcloud-java-nio/src/main/java/com/google/gcloud/storage/contrib/nio/CloudStorageConfiguration.java#L34) and [here](https://github.com/GoogleCloudPlatform/gcloud-java/blob/gcs-nio/gcloud-java-contrib/gcloud-java-nio/src/main/java/com/google/gcloud/storage/contrib/nio/CloudStorageConfiguration.java#L53)).  There are also a few scattered in CloudStorageFileAttributesTest and CloudStoragePathTest.

File: gcloud-java-contrib/gcloud-java-nio/src/main/java/com/google/gcloud/storage/contrib/nio/CloudStoragePath.java [hunk 3]
```diff
@@ -273,9 +277,9 @@ public int compareTo(Path other) {
   public boolean equals(@Nullable Object other) {
     return this == other
         || other instanceof CloudStoragePath
-        && Objects.equals(bucket(), ((CloudStoragePath) other).bucket())
-        && Objects.equals(toRealPathInternal(false),
-            ((CloudStoragePath) other).toRealPathInternal(false));
+            && Objects.equals(bucket(), ((CloudStoragePath) other).bucket())
```
Comment (aozarov) [inline gcloud-java-contrib/gcloud-java-nio/src/main/java/com/google/gcloud/storage/contrib/nio/CloudStoragePath.java]: I mentioned it in the prior PR but I don't remember an answer (could be my fault).
FindBugs complains that setting `other` as `@Nullable` is wrong as it is not compatible with the method it overrides.
Clearly, it is a harmless message but I would still like to raise (I am actually not sure what is the recommendation for it).

Comment (jart) [inline gcloud-java-contrib/gcloud-java-nio/src/main/java/com/google/gcloud/storage/contrib/nio/CloudStoragePath.java]: Sorry I forgot about that one. Thanks for bringing it up. I've fixed this in #715. I also remembered what you said about tests throwing exceptions, and fixed that too :)

File: gcloud-java-contrib/gcloud-java-nio/src/main/java/com/google/gcloud/storage/contrib/nio/UnixPath.java (hunk 1 above)
Comment (jart) [inline gcloud-java-contrib/gcloud-java-nio/src/main/java/com/google/gcloud/storage/contrib/nio/UnixPath.java]: Changed to "path" so it doesn't need code font in #715.

File: gcloud-java-contrib/gcloud-java-nio/src/main/java/com/google/gcloud/storage/contrib/nio/CloudStorageObjectImmutableException.java (hunk 2 above)
Comment (jart) [inline gcloud-java-contrib/gcloud-java-nio/src/main/java/com/google/gcloud/storage/contrib/nio/CloudStorageObjectImmutableException.java]: Good catch. I grepped and made sure I got all instances of this in #715.

**Pull request merged**